testutil/directory.go

```diff
@@ -143,3 +146,43 @@ func DirSize(path string) (int64, error) {
 	})
 	return size, err
 }
+
+// DirHash returns a hash of all files attribites and their content within a directory.
+func DirHash(path string) ([]byte, error) {
+	hash := md5.New()
```

Comment:
We can change it to something better, like `sha256`.

Reply:
updated